PyInstaller/utils/hooks/qt.py

```diff
@@ -396,7 +396,8 @@ def get_qmake_path(version=''):
     "enginio":                  (None,                     None,               ),
     "qt5gamepad":               (None,                     None,               "gamepads"),
     # Note: The ``platformthemes`` plugin is for Linux only, and comes from earlier PyInstaller code in ``hook-PyQt5.QtGui.py``. The ``styles`` plugin comes from the suggestion at https://github.com/pyinstaller/pyinstaller/issues/2156.
-    "qt5gui":                   (".QtGui",                 "qtbase",           "accessible", "iconengines", "imageformats", "platforms", "platforminputcontexts", "platformthemes", "styles"),
+    # ``xcbglintegrations`` and ``egldeviceintegrations`` were added manually for linux
+    "qt5gui":                   (".QtGui",                 "qtbase",           "accessible", "iconengines", "imageformats", "platforms", "platforminputcontexts", "platformthemes", "styles", "xcbglintegrations", "egldeviceintegrations"),
```

Comment:
Maybe should help here?
```suggestion
    "qt5gui":                   (".QtGui",                 "qtbase",           "accessible", "iconengines", "imageformats", "platforms", "platforminputcontexts", "platformthemes", "styles", "xcbglintegrations", "egldeviceintegrations"),  # noqa
```

Reply:
It would, but I'm not sure whether we should really bother. That whole dict is in violation of linter rules, so we would need a `noqa` in each line. It might be easier and saner to simply ignore the linter in this case.

Reply:
Fix this one line, seeing as you've already edited it. Otherwise looks good